types/parse/ts3.7/index.d.ts

```diff
@@ -824,8 +847,10 @@ namespace Parse {
         setPassword(password: string, options?: SuccessFailureOptions): boolean;
         getSessionToken(): string;
 
-        linkWith(user: User, authData: AuthData, options: FullOptions): Promise<User>;
-        _linkWith(provider: any, options: { authData?: AuthData }, saveOpts?: FullOptions): Promise<User>;
+        linkWith: (provider: string | AuthProvider, options: { authData?: AuthData }, saveOpts?: FullOptions) => Promise<this>;
+        _linkWith: (provider: string | AuthProvider, options: { authData?: AuthData }, saveOpts?: FullOptions) => Promise<this>;
```

Comment:
I think this was already deprecated since 2.9.0. Why add it here?

Reply:
Since it hasn't been removed yet I thought it ought to be there until it is removed, but I can remove `_linkWith` if that is preferred.

Reply:
Makes sense, but since the upgrade 2.9.0, they started to deprecate it and no one seems to bother about it thats why I thought we ought to remove it :smile: